infra/monitoring/README.md

```diff
@@ -0,0 +1,7 @@
+# Monitoring Infra
+
+To setup an instance of the monitoring infrastructure:
+- create a GCP project and set it as L12N_GCP_PROJECT_ID
+- choose a GCP region and set it as L12N_GCP_REGION
+- before deploying, you must run `l12n monitoring.login` and authenticate
+  yourself with an email that has access to the project ID specified above
```

Comment:
add description of use of GCP and the need to deploy both:
* l12n monitoring.deploy (GCP infra) and 
* l12n deploy --stage monitoring (plugin data collection)